sqlite.zig

```diff
@@ -590,6 +590,34 @@ pub const Db = struct {
     pub fn savepoint(self: *Self, name: []const u8) Savepoint.InitError!Savepoint {
         return Savepoint.init(self, name);
     }
+
+    /// This is a convenience function to run statements that do not need
+    /// bindings to values, but have multiple commands inside.
+    ///
+    /// Exmaple: 'create table a(); create table b();'
+    pub fn runMulti(self: *Self, comptime query: []const u8, options: QueryOptions) !void {
+        var new_options = options;
+        var sql_tail_ptr: ?[*:0]const u8 = null;
+        new_options.sql_tail_ptr = &sql_tail_ptr;
+
+        while (true) {
+            // continuously prepare and execute (dynamically as there's no
+            // values to bind in this case)
+            var stmt: DynamicStatement = undefined;
+            if (sql_tail_ptr != null) {
+                const new_query = std.mem.span(sql_tail_ptr.?);
+                if (new_query.len == 0) break;
+                stmt = try self.prepareDynamicWithDiags(new_query, new_options);
+            } else {
+                stmt = try self.prepareDynamicWithDiags(query, new_options);
+            }
+
+            defer stmt.deinit();
+            try stmt.exec(new_options, .{});
+
+            if (sql_tail_ptr == null) break;
```

Comment:
Is there a case where this can happen ? As far as I can tell in my testing the exit condition is always the `if (new_query.len == 0)` above.